lib/travis/api/v3/models/build.rb

```diff
@@ -80,5 +80,17 @@ def branch_name=(value)
     def clear_debug_options!
       jobs.each { |j| j.update_attribute(:debug_options, nil) }
     end
+
+    def log_complete
+      archived = []
```

Comment:
Maybe this could be shortened a bit, e.g. like

```ruby
    def log_complete
      archived = jobs.each_with_object([]) do |j,a|
        a << Travis::RemoteLog.find_by_job_id(j.id).archived?
      end
      archived.all? true
    end
```